reports/ISCNVisualize.Rmd

```diff
@@ -31,6 +34,54 @@ ISCN3 <- SoilDataR::processData_ISCN3(dir='../repoData/ISCN_3/')
 ```
 
 
+#Observation time
+```{r obsTime}
+location.df <- unique(ISCN3$field[, c('lat', 'long', 'observation_date', 
+                                       'state', 'country')])
+location.df$observation_date <- as.character(location.df$observation_date)
+
+location.df <- location.df %>%
+  separate(observation_date, c('monthStr', 'dayStr', 'yearStr'), 
+           remove=FALSE, fill='left') %>%
+  mutate(year=ifelse(as.numeric(yearStr) < 20, as.numeric(yearStr) + 2000,
```

Comment:
`ifelse` in R can return _very_ unexpected results (it strips attributes; can change class; and other ugliness). If this is working for you here, OK, but I'd recommend staying away from it in the future.

Use `dplyr::if_else` instead.

Reply:
Actually dplyr::case_when looks like an even better option. Done here 79055e075ec0b9c7b25f7f072353abd7f95848a8